package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apptentive-react-native",
-  "version": "6.1.1",
+  "version": "6.1.2",
```

Comment:
I see this version is using 6.2.0 iOS SDK. Would that include multi user support? Should this release go with a major version change like 6.2?

Reply:
I don't think we should add multi-user support until Android has it as well, since it would only work on one platform.

Reply:
Ohhk. I thought iOS 6.2 would support multi user.